var/spack/repos/builtin/packages/ldsc/package.py

```diff
@@ -10,7 +10,7 @@ class Ldsc(PythonPackage):
     """ldsc is a command line tool for estimating heritability and genetic correlation from
```

Comment:
Python package without `py-` prefix. The class name should be `PyLdsc` and the directory name should be `py-ldsc`

Reply:
This is allowed, particularly when it is a stand alone tool and not a library.